docs/working_docs/output.md

```diff
@@ -73,7 +73,15 @@ Sample output:
               "ids": [
                 "GHSA-c3h9-896r-86jm",
                 "GO-2021-0053"
-              ]
+              ],
+              // Call stack analysis is done using the `--experimental-call-analysis` flag
+              // and result is matched against data provided by the advisory to check if
+              // affected code is actually being executed.
+              "analysis": {
```

Comment:
should we prefix this with "experimental-" for now?

Reply:
yep, changed to experimentalAnalysis